.github/workflows/ci.yml

```
@@ -0,0 +1,77 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  should-skip:
+    continue-on-error: true
+    runs-on: ubuntu-18.04
+    # Map a step output to a job output
+    outputs:
+      should-skip-job: ${{steps.skip-check.outputs.should_skip}}
+    steps:
+      - id: skip-check
+        uses: fkirc/skip-duplicate-actions@v2.1.0
+        with:
+          github_token: ${{github.token}}
+
+  ci:
+    needs: should-skip
+    if: ${{needs.should-skip.outputs.should-skip-job != 'true'}}
```

```suggestion
    if: ${{needs.should-skip.outputs.should-skip-job != 'true' || github.ref == 'refs/heads/main'}}
```

don't skip in main branch from https://github.com/videojs/video.js/pull/7169